test/commandLine.test.js

```diff
@@ -31,7 +31,7 @@ describe('CommandLineManager', function () {
 
     it('parse proxy-host and proxy-port params', function () {
       sinon.stub(console, 'log');
-      argv = argv.concat(['--proxy-host', 'host', '--proxy-port', '9687']);
+      argv = argv.concat(['--proxy-host', 'http://host', '--proxy-port', '9687']);
       CommandLineManager.processArgs(argv);
       console.log.restore();
       expect(RdGlobalConfig.proxy.host).to.eql('host');
```

Comment:
We can add another test case in which we have both the scenarios:
1. with the protocol
2. without the protocol

Reply:
Done.